src/DatePicker/DatePickerDialog.js

```diff
@@ -136,7 +138,7 @@ class DatePickerDialog extends Component {
           repositionOnUpdate={true}
           open={open}
           onRequestClose={this.handleRequestClose}
-          style={styles.dialogBodyContent}
+          style={Object.assign(styles.dialogBodyContent, containerStyle)}
```

Comment:
It looks like that `<div {...other} ref="root">` and the duplication of `{...other}` were added back in https://github.com/callemall/material-ui/pull/3739/files#r61083381.

There was some conversation there, but nothing that explains why the extra div was needed when they changed from `DatePickerInline` to `Popover`. From that conversation, it sounds like a somewhat larger refactoring of this component may be worthwhile as another PR.